boards/arm/stm32h747i_disco/stm32h747i_disco_m7.yaml

```diff
@@ -6,8 +6,8 @@ toolchain:
   - zephyr
   - gnuarmemb
   - xtools
-ram: 512
-flash: 1024
+ram: 1024
+flash: 2048
```

Comment:
This limitation was done on purpose as this target is dual code and flash and ram should be shared between both cores.
Even if in practice, these fields are used only by twister, I prefer we keep current configuration for coherency with other parts of board configuration.
We can review once these settings are preventing testing on some tests/samples, but I think existing settings should be sufficient for some time on that front.

Reply:
ok, reverted the changes.